sematic/examples/add/__main__.py

```diff
@@ -11,10 +11,11 @@
     parser.add_argument("--cloud", action="store_true", default=False)
     parser.add_argument("--detach", action="store_true", default=False)
     parser.add_argument("--rerun-from", default=None)
+    parser.add_argument("--debug", default=False, action="store_true")
```

Comment:
You can use the testing pipeline, don't you know?

Reply:
Ah yeah 😬